packages/shared-ux/page/analytics_no_data/src/analytics_no_data_page.component.test.tsx

```diff
@@ -7,20 +7,31 @@
  */
 
 import React from 'react';
+import { act } from 'react-dom/test-utils';
+
 import { mountWithIntl } from '@kbn/test-jest-helpers';
-import { KibanaNoDataPage } from '@kbn/shared-ux-components';
+import { KibanaNoDataPage } from '@kbn/shared-ux-page-kibana-no-data';
+
 import { AnalyticsNoDataPage } from './analytics_no_data_page.component';
+import { AnalyticsNoDataPageProvider } from './services';
+import { getJestServices } from './mocks';
```

Comment:
I would not call this `jestServices`, but something more generic, like `mockServices` or `testServices`.